Assets/Scenes/Disclaimer.unity

```diff
@@ -249,7 +249,7 @@ MonoBehaviour:
     m_HorizontalOverflow: 0
     m_VerticalOverflow: 0
     m_LineSpacing: 1
-  m_Text: 'v0.6.5
+  m_Text: 'v0.6.6
```

Comment:
Could you please revert this change? I only do that when the next version is ready to be distributed!